query03.sql

```diff
@@ -0,0 +1,48 @@
+/*
+  Using the Philadelphia Water Department Stormwater Billing Parcels dataset,
+  pair each parcel with its closest bus stop. The final result should give the
+  parcel address, bus stop name, and distance apart in meters. Order by
+  distance (largest on top).
+*/
+/*create index for*/
+/*similar answer https://qa.1r1g.com/sf/ask/3398603561/*/
+DROP INDEX IF EXISTS phl_pwd_parcels_the_geom_idx;
+CREATE index phl_pwd_parcels_the_geom_idx
+	on phl_pwd_parcels
+	using GiST(st_transform(the_geom, 32129));
+
+DROP INDEX IF EXISTS septa_bus_stops_the_geom_idx;
+create index septa_bus_stops_the_geom_idx
+    on septa_bus_stops
+    using GiST(st_transform(the_geom, 32129));
+
+     select
+     	a.stop_name,
+     	dis.address,
+     	dis.distance_m
+     from septa_bus_stops a
```

Comment:
Note this gives the closest address to each bus stop rather than the closest bus stop to each address.